.github/vale/Vocab/Backstage/accept.txt

```diff
@@ -28,6 +28,7 @@ bool
 boolean
 builtins
 callout
+catalogPath
```

Comment:
you can use code formatting for these at places where vale complained about them like

```
`catalogPath`
```

the vale will accept it regardless.